.devops/repo/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use clap::{Parser, ValueEnum};
+
+mod commands;
+mod utils;
+
+use commands::{build, bump, fmt, test};
+use utils::{read_applications, repo_root};
+
+#[derive(Parser)]
+#[command(about = "Utility crate for maintaining the repository")]
+struct Cli {
+    #[arg(value_enum)]
+    command: Mode,
+}
+
+#[derive(Clone, ValueEnum)]
+enum Mode {
+    /// Build the Sway contracts for each project
+    Build,
+    /// Bump each project from its current `fuel-toolchain.toml` to the one in this repository
+    Bump,
+    /// Format the Sway and Rust files in each project
+    Fmt,
+    /// Run the Rust tests for each project
+    Test,
+}
+
+fn main() {
```

Comment:
That also means we return `anyhow::Result` here:

```suggestion
fn main() -> Result<()> {
```